File: Python/110A. Nearly Lucky Number.py
n = input()
d = 0
i = 0
while i<len(n):
    if n[i] == '4' or n[i] == '7':
        d += 1
    i += 1
if d == 4 or d == 7:
    print("YES")
else:
    print("NO")

File: C++/110A. Nearly Lucky Number.cpp
#include<bits/stdc++.h>
using namespace std;
int main()
{
    long long n, i, m, d=0;
    cin>>n;
    for(i=0; n!=0; i++){
        m = n%10;
        n = n/10;
        if(m==4||m==7){
            d++;
        }
    }
    if(d==4||d==7){
        cout<<"YES";
    }
    else{
        cout<<"NO";
    }
    return 0;
}